hack/allure-report.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -ex
+
+CE=${CE:-podman}
```

Comment:
can we come up with some better name and document it perhaps on the level of Makefile? How about `CONTAINER_RUNTIME`?

Reply:
Good idea